selectinf/randomized/posterior_inference.py

```diff
@@ -31,39 +31,46 @@ def __init__(self,
                  query,
                  observed_target,
                  cov_target,
-                 cov_target_score,
+                 regress_target_score,
+                 dispersion,
```

Comment:
About `dispersion`.... besides   for selectiveMLE (which I've removed as an argument) why does posterior inference require it -- shouldn't the `cov_target` have its dispersion taken into account already?